src/test/java/StringManipulationTest.java

```diff
@@ -232,6 +232,24 @@ public void testRestoreString2()
         assertEquals(restoreString, "five four");
 
     }
+    
+    // This new test is in response to Alvin Bautista's GitHub code review:
```

Comment:
@ACBxBC, this is the new test in response to your comment on whitespaces. Check it out and let me know what you think!